sdk/lib/convert/latin1.dart

```diff
@@ -63,12 +63,50 @@ class Latin1Codec extends Encoding {
 }
 
 /// This class converts strings of only ISO Latin-1 characters to bytes.
+///
+/// Example:
+/// ```dart
+/// const String sample = 'àáâãäå';
+///
+/// const Latin1Encoder latin1Encoder = Latin1Encoder();
+/// final List<int> encoded = latin1Encoder.convert(sample);
+/// print(encoded); // [224, 225, 226, 227, 228, 229]
+/// ```
 class Latin1Encoder extends _UnicodeSubsetEncoder {
   const Latin1Encoder() : super(_latin1Mask);
 }
 
 /// This class converts Latin-1 bytes (lists of unsigned 8-bit integers)
 /// to a string.
+///
+/// Example:
+/// ```dart
+/// const Latin1Decoder latin1Decoder = Latin1Decoder();
+/// const List<int> encodedBytes = [224, 225, 226, 227, 228, 229];
+/// final String decoded = latin1Decoder.convert(encodedBytes);
+/// print(decoded); // àáâãäå
+///
+/// // Hexadecimal values as source
+/// const List<int> hexBytes = [0xe0, 0xe1, 0xe2, 0xe3, 0xe4, 0xe5];
+/// final String decodedHexBytes = latin1Decoder.convert(hexBytes);
+/// print(decodedHexBytes); // àáâãäå
+/// ```
+///
+/// If `bytes` contains values that are not in the range 0 .. 255, the decoder
+/// throws [FormatException].
```

Comment:
nit: should probably have an "a"

```suggestion
/// throws a [FormatException].
```